ensembl_dbcopy/static/js/multiselect.js

```diff
@@ -47,7 +47,7 @@ function hostStringToDetails(string) {
 }
 
 function hostDetailsToString(details) {
-  return details.name + ":" + details.port;
+  return details.name + ":" + details.port ;
```

Comment:
If `hostDetailsToString` is not used anymore, please remove it.

Reply:
hostDetailsToString is used better to keep it